src/components/AvatarWithDisplayName.js

```diff
@@ -17,6 +17,10 @@ import compose from '../libs/compose';
 import * as OptionsListUtils from '../libs/OptionsListUtils';
 import Text from './Text';
 import * as StyleUtils from '../styles/StyleUtils';
+import {getChatRoomSubtitleLink} from '../libs/ReportUtils';
```

Comment:
seems like this line was not passed to lint. check the syntax again.

Reply:
PR is in a draft state and not ready to review just yet 🙂